commands/assets/assets.go

```diff
@@ -0,0 +1,43 @@
+package assets
+
+import (
+	"fmt"
+
+	"github.com/netlify/netlifyctl/commands/middleware"
+	"github.com/netlify/netlifyctl/configuration"
+	"github.com/spf13/cobra"
+)
+
+func Setup(middlewares []middleware.Middleware) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "assets",
+		Short: "List assets attached to a site",
+		Long:  "List assets attached to a site",
+	}
+	cmd.PersistentFlags().StringP("site-id", "s", "", "site id")
+
+	cmd.AddCommand(setupAddCommand(middlewares))
+	cmd.AddCommand(setupInfoCommand(middlewares))
+
+	return middleware.SetupCommand(cmd, listAssets, middlewares)
+}
+
+func siteIdForCommand(cmd *cobra.Command) (string, error) {
+	siteId := cmd.Flag("site-id").Value.String()
+
+	var configFile = cmd.Root().Flag("config").Value.String()
```

Comment:
instead of this you should just be able to access the `configFile` specified in the root_cmd